src/AltinnCore/Common/Configuration/ServiceRepositorySettings.cs

```diff
@@ -185,12 +185,12 @@ public class ServiceRepositorySettings
         /// <summary>
         /// Gets or sets React file name
         /// </summary>
-        public string ReactAppFileName { get; set; } = "react-app.js";
+        public string ReactAppFileName { get; set; } = "runtime.js";
```

Comment:
Don't know how much impact the changing of name of this will have, but probably rename it to `RuntimeAppFileName`?